package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "telemetryclient-team-services-extension",
+  "version": "1.0.8",
+  "description": "Track telemetry data for your Team Services extension",
+  "homepage": "https://github.com/ALM-Rangers/Application-Insights-Client",
```

Comment:
What is the difference between https://github.com/ALM-Rangers/Application-Insights-Client and the repository Uri https://github.com/ALM-Rangers/telemetryclient-team-services-extension